docker/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3"
+services:
+  web-server:
+    image: emission-server
+    depends_on:
+      - db
+    environment:
+      - DB_HOST=db
+      - WEB_SERVER_HOST=0.0.0.0
+    deploy:
+      replicas: 1
+      restart_policy:
+        condition: on-failure
+    ports:
+      #This is a default port mapping. In production you might want to use 80:8080,
+      - "8080:8080"
+    networks:
+      - emission
+  db:
+    image: mongo:latest
+    deploy:
+      replicas: 1
+      restart_policy:
+        condition: on-failure
+    ports:
+      - "27017:27017"
```

Comment:
this is only if you want to access the DB from the host. Add a note to that effect?